packages/desktop-client/src/components/reports/DateRange.tsx

```diff
@@ -34,43 +36,59 @@ export function DateRange({ start, end, type }: DateRangeProps): ReactElement {
   } else {
     return (
       <Text style={{ ...styles.mediumText, color: theme.errorText }}>
-        There was a problem loading your date range
+        <Trans>There was a problem loading your date range</Trans>
       </Text>
     );
   }
 
+  const formattedStartDate = d.format(startDate, 'MMM yyyy');
+  const formattedEndDate = d.format(endDate, 'MMM yyyy');
+  let typeOrFormattedEndDate: string;
+
+  if (type && ['budget', 'average'].includes(type)) {
+    typeOrFormattedEndDate = type;
+  } else {
+    typeOrFormattedEndDate = formattedEndDate;
+  }
+
   let content: string | ReactElement;
   if (['budget', 'average'].includes(type || '')) {
     content = (
       <div>
-        Compare {d.format(startDate, 'MMM yyyy')} to{' '}
-        {type === 'budget' ? 'budgeted' : 'average'}
-      </div>
-    );
-  } else if (startDate.getFullYear() !== endDate.getFullYear()) {
-    content = (
-      <div>
-        {type && 'Compare '}
-        {d.format(startDate, 'MMM yyyy')}
-        {type ? ' to ' : ' - '}
-        {['budget', 'average'].includes(type || '')
-          ? type
-          : d.format(endDate, 'MMM yyyy')}
+        <Trans
+          values={{
+            startDate: formattedStartDate,
+            endDate: typeOrFormattedEndDate,
+          }}
+        >
+          Compare {{ startDate }} to {{ endDate }}
+        </Trans>
       </div>
     );
-  } else if (startDate.getMonth() !== endDate.getMonth()) {
+  } else if (
+    startDate.getFullYear() !== endDate.getFullYear() ||
+    startDate.getMonth() !== endDate.getMonth()
+  ) {
     content = (
       <div>
-        {type && 'Compare '}
-        {d.format(startDate, 'MMM yyyy')}
-        {type ? ' to ' : ' - '}
-        {['budget', 'average'].includes(type || '')
-          ? type
-          : d.format(endDate, 'MMM yyyy')}
+        {type ? (
+          <Trans
+            values={{
+              formattedStartDate,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could we use `startDate` for the sake of consistency?
```suggestion
              startDate: formattedStartDate,
```